R/tags.R

```diff
@@ -958,6 +954,49 @@ flattenTags <- function(x) {
     flattenTags(as.tags(x))
   }
 }
+# This method should be just like `flattenTags()`, except the final `else` will return `list(x)`, rather than calling `flattenTags(as.tags(x))`.
+# By not calling `as.tags(x)`, tagFunctions are not evaluated and other items are not converted.
+flattenTagsRaw <- function(x) {
```

Comment:
It's kind of unclear to me exactly what this function is supposed to do. It's called `flattenTagsRaw`, but it appears to wrap tags, tag-envs, and strings in lists, and it recurses into to lists/taglists.

So, this is like `flattenTags()`, except it accepts tag-envs, and it doesn't invoke tagFunctions?

Reply:
👍